configs/components/gcc.rb

```diff
@@ -1,6 +1,6 @@
 component "gcc" do |pkg, settings, platform|
   # Source-Related Metadata
-  if platform.name =~ /fedora-f24|fedora-f25|ubuntu-16\.10/
+  if platform.name =~ /fedora-f24|fedora-f25|ubuntu-16\.04-ppc64el|ubuntu-16\.10/
```

Comment:
A pl-gcc rev for a platform requires rebuilding and re-shipping everything else in pl-build-tools - but I'd describe it as a low-risk rebuild that can be done as a 3-point background task (just to account for context switching load). If you want to go forward with this, I'm not opposed to it.